src/components/data/data-utils.js

```diff
@@ -491,7 +493,9 @@ export const EntityEditor = ({ entityType, entityName, attributeName, attributeV
   ))
   const [editType, setEditType] = useState(() => Utils.cond(
     [initialIsReference, () => 'reference'],
-    [(initialIsList ? attributeValue.items[0] : attributeValue) === undefined, () => 'string'],
+    // explicit double-equal to check for null and undefined, since entity attribute lists can contain nulls
+    // eslint-disable-next-line eqeqeq
+    [(initialIsList ? attributeValue.items[0] : attributeValue) == undefined, () => 'string'],
```

Comment:
here and above, we were previously comparing `null === undefined`, which is false, and we fell down to the default condition `typeof attributeValue` … and `typeof(null)` is `object`. When the UI attempted to edit something it though was an object but was actually null, the UI broke. Now, by changing `===` to `==`, this condition returns true because `null == undefined`, and we treat nulls as strings. This allows the UI to edit them correctly.